#+hugo_base_dir: ../

#+TITLE: Gregory W. Schwartz
#+AUTHOR: Gregory W. Schwartz
#+EMAIL: gregory.schwartz@uhnresearch.ca
#+LANGUAGE: en

#+HTML_HEAD: <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.5.1/jquery.js"></script>
#+HTML_HEAD: <script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

* Schwartz Lab
:PROPERTIES:
:EXPORT_HUGO_SECTION:
:END:

** Statement

Life is a complex system of interconnecting networks: from the transcriptome to
the proteome, from a single cell to clonal populations, each layer of
information can be interpreted as a network where each molecule, cell, or tissue
in some way affects the overall system. *Systems biology* attempts to take into
account as many of these levels as possible in order to answer biological
questions pertaining to not only the organization of life but also separating
signal from noise to discover key components of disease or adverse clinical
outcomes. While sampling technologies give partial insight to these issues, the
other half of the solution, namely mathematical and computational analyses of
biological data, are needed more than ever in this high-throughput era.

The need for such analyses drives my research. In my efforts to elucidate the
relationships between genotype and phenotype, clonal diversity and selection,
and integration of multiple modalities of information (transcriptome, proteome,
phosphoproteome, etc.), I have developed algorithms, programs, and mathematical
measures.

More specifically, I am interested in clonal behavior and evolution, pattern
recognition of the interactome through integration, network structure
similarity, data visualization, and everything in-between.

* Research
:PROPERTIES:
:EXPORT_HUGO_SECTION:
:EXPORT_FILE_NAME: research
:END:

Chemotherapy and targeted drugs are effective treatments against cancer, *but
commonly lead to treatment-resistant cancer cells*. Cancer heterogeneity is known
to play a significant role in contributing towards this resistance. In order to
overcome treatment-resistance in cancer, our goal is to develop new
computational methods and algorithms to quantify and visualize cancer
heterogeneity to better understand the underlying mechanisms of resistance. To
accomplish this goal, we are creating new integrative tools at different scales
(such as between cancer subtypes or between individual cells within a single
cancer population) and data modalities (such as the transcriptome and the
epigenome) to guide therapy.

Some past examples of these initiatives as part of the [[https://faryabilab.com/][Faryabi Lab]] include:

#+BEGIN_EXPORT html
<div class="clearfix">

<img src=/img/too-many-cells.svg class="pull-left" style="margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

Single-cell analyses enable measurement of the transcriptome, chromatin
accessibility, and more at single-cell resolution. However, grouping similar
cells at different resolution cutoffs severely limits the exploration of both
common and rare heterogeneous populations. We have developed =TooManyCells=, a
suite of novel tools focused on simultaneous visualization of single-cell clade
relationships at all resolutions for single-cell RNA sequencing (scRNA-seq)
data.

#+BEGIN_EXPORT html
</div>

<br></br>

<div class="clearfix">

<img src=/img/too-many-peaks-graphical-abstract.svg class="pull-left" style="margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

While single-cell RNA sequencing can reveal much about the transcriptomic state
of individual cells, these states are largely set by the interaction of
transcription factors with regulatory elements within accessible chromatin
regions. To understand the relationship between cells with similar or unique
chromatin states, we developed =TooManyPeaks=, part of the =TooManyCells= suite,
for end-to-end analysis of single-cell assay for transposase-accessible
chromatin using sequencing (scATAC-seq) data.

#+BEGIN_EXPORT html
</div>

<br></br>


<div class="clearfix">

<img src=/img/heatitup.svg class="pull-left" style="margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

#+attr_html: :class clearfix
Poor prognosis of patients with acute myeloid leukemia can be predicted based on
the presence of internal tandem duplications (ITDs) in Fms-like tyrosine kinase
3 (/FLT3/). We discovered new classes of /FLT3/-ITDs that predict patient
outcomes by creating =HeatITup=, an algorithm that identifies, characterizes,
and visualizes these classes of /FLT3/-ITDs.

#+BEGIN_EXPORT html
</div>

<br></br>

<div class="clearfix">

<img src=/img/integreat.svg class="pull-left" style="margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

While we can measure different modalities, such as the transcriptome and
proteome, it is difficult to analytically relate each level of information in a
systematic manner. We developed =inteGREAT=, an algorithm to use integration and
differential integration between conditions to identify biomarkers across
modalities.

#+BEGIN_EXPORT html
</div>

<br></br>
#+END_EXPORT

* Publications
:PROPERTIES:
:EXPORT_HUGO_SECTION:
:EXPORT_FILE_NAME: publications
:END:

#+BEGIN_EXPORT html
<style>
li:not(:last-child) {
    margin-bottom: 2%;
}
</style>
#+END_EXPORT

- Fasolino M*, *Schwartz GW**, Patil AR, Mongia A, Golson ML, Wang
  YJ, Morgan A, Liu C, Schug J, Liu J, Wu M, Traum D, Kondo A, May CL, Goldman
  N, Wang W, Feldman M, Moore JH, Japp AS, Betts MR, Faryabi RB, Naji A,
  Kaestner KH, Vahedi G. Single-cell multi-omics analysis of human pancreatic
  islets reveals novel cellular states in type 1 diabetes /Nat. Metab./ *4* 284-299 (2022).
  [[http://www.nature.com/articles/s42255-022-00531-x][doi:10.1038/s42255-022-00531-x]]. *Authors contributed equally.
- <<tooManyPeaks>> *Schwartz GW*, Zhou Y, Petrovic J, and Faryabi RB. TooManyPeaks
    identifies drug-resistant-specific regulatory elements from single-cell
    leukemic epigenomes. /Cell Rep./ *36* (2021).
    [[https://doi.org/10.1016/j.celrep.2021.109575][doi:10.1016/j.celrep.2021.109575]]
- <<tooManyCells>> *Schwartz GW*, Petrovic J, Fasolino M, Zhou Y, Cai S, Xu L,
  Pear WS, Vahedi G, and Faryabi RB. TooManyCells identifies and visualizes
  relationships of single-cell clades. /Nat. Methods/ *17* 405–413 (2020).
  [[https://doi.org/10.1038/s41592-020-0748-5][doi:10.1038/s41592-020-0748-5]]
- Rome KS, Stein, SJ, Kurachi M, Petrovic J, *Schwartz GW*, Mack EA, Uljon S, Wu
  WW, DeHart AG, McClory SE, Xu L, Gimotty PA, Blacklow SC, Faryabi RB, Wherry
  EJ, Jordan MS, Pear WS. Trib1 regulates T cell differentiation during chronic
  infection by restraining the effector program. /J. Exp. Med./ *217* e20190888
  (2020).
  [[https://doi.org/10.1084/jem.20190888][doi:10.1084/jem.20190888]]
- Petrovic J, Zhou Y, Fasolino M, Goldman N, *Schwartz GW*, Mumbach MR, Nguyen SC,
  Rome KS, Sela Y, Zapataro Z, Blacklow SC, Kruhlak MJ, Shi J, Aster JC, Joyce
  EF, Little SC, Vahedi G, Pear WS, Faryabi RB. Oncogenic Notch Promotes
  Long-Range Regulatory Interactions within Hyperconnected 3D Cliques. /Mol.
  Cell./ *73* 1174-1190.e12. (2019).
  [[https://www.cell.com/molecular-cell/fulltext/S1097-2765(19)30006-1?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276519300061%3Fshowall%3Dtrue][doi:10.1016/j.molcel.2019.01.006]]
- <<flt3Class>> *Schwartz GW*, Manning BS, Zhou Y, Velu PD,
  Bigdeli A, Astles R, Lehman AW, Morrissette JJD, Perl AE, Li M, Carroll M, and
  Faryabi RB. Classes of ITD predict outcomes in AML patients treated with FLT3
  inhibitors. /Clin. Cancer Res./ (2019).
  [[http://clincancerres.aacrjournals.org/content/25/2/573][doi:10.1158/1078-0432.CCR-18-0655]]
  Commentary: Prognostic Models Turn the Heat(IT)up on FLT3^{ITD}-Mutated AML.
  [[http://clincancerres.aacrjournals.org/content/25/2/460?iss=2][doi:10.1158/1078-0432.CCR-18-3146]]
- <<integration>> *Schwartz GW*, Petrovic J, Zhou Y, and Faryabi RB.
  Differential integration of transcriptome and proteome identifies pan-cancer
  prognostic biomarkers. /Front. Genet./ *9*, 205 (2018).
  [[https://www.frontiersin.org/articles/10.3389/fgene.2018.00205/full][doi:10.3389/fgene.2018.00205]]
- <<bCellAtlas>> Meng W*, Zhang B*, *Schwartz GW*, Rosenfeld AM, Ren D, Thome J
  JC, Carpenter DJ, Matsuoka N, Lerner H, Friedman AL, Granot T, Farber DL,
  Shlomchik MJ, Hershberg U and Luning Prak ET. An atlas of B cell clonal
  distribution in the human body. /Nat. Biotechnol./ *35*, 879-884 (2017).
  [[https://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.3942.html][doi:10.1038/nbt.3942]]
- Antell GC, Dampier W, Aiamkitsumrit B, Nonnemacher MR, Jacobson JM, Pirrone V,
  Zhong W, Kercher K, and Passic S, Williams JW, *Schwartz G*, Hershberg U,
  Krebs FC, and Wigdahl B. Utilization of HIV-1 envelope V3 to identify X4- and
  R5-specific Tat and LTR sequence signatures. /Retrovirology/ *13*, 32 (2016).
  [[http://retrovirology.biomedcentral.com/articles/10.1186/s12977-016-0266-9][doi:10.1186/s12977-016-0266-9]]
- *Schwartz GW*, Shauli T, Linial M, and Hershberg U. Serine substitutions are
  linked to codon usage and differ for variable and conserved protein regions.
  /Sci. Rep./ *9*, 1 (2019).
  [[http://www.nature.com/articles/s41598-019-53452-3][doi:10.1038/s41598-019-53452-3]]
- <<clumpinessPaper>> *Schwartz GW*, Shokoufandeh A, Ontañón S, and Hershberg U.
  Using a novel clumpiness measure to unite data with metadata: finding common
  sequence patterns in immune receptor germline V genes. /Pattern Recogn. Lett./
  *74*, 24-29 (2016). [[http://www.sciencedirect.com/science/article/pii/S0167865516000234][doi:10.1016/j.patrec.2016.01.011]]
- Meng W, Jayaraman S, Zhang B, *Schwartz GW*, Daber RD, Hershberg U,
  Garfall AL, Carlson CS and Luning Prak ET. Trials and tribulations with VH
  replacement. /Front. Immunol./ *5*, 10 (2014). [[http://www.frontiersin.org/Journal/10.3389/fimmu.2014.00010/abstract][doi:10.3389/fimmu.2014.00010]]
- <<diversitySelectionPaper>> *Schwartz GW*, Hershberg U. Germline Amino Acid
  Diversity in B Cell Receptors is a Good Predictor of Somatic Selection
  Pressures. /Front. Immunol./ *4*, 357 (2013). [[http://www.frontiersin.org/Journal/10.3389/fimmu.2013.00357/abstract][doi:10.3389/fimmu.2013.00357]]
- <<diversityPaper>> *Schwartz GW*, Hershberg U. Conserved variation:
  identifying patterns of stability and variability in BCR and TCR V genes with
  different diversity and richness metrics. /Phys. Biol./ *10*, 035005 (2013).
  [[http://iopscience.iop.org/1478-3975/10/3/035005/][doi:10.1088/1478-3975/10/3/035005]]

#+BEGIN_EXPORT html
<br></br>
#+END_EXPORT

* Software
:PROPERTIES:
:EXPORT_HUGO_SECTION:
:EXPORT_FILE_NAME: software
:END:

#+BEGIN_EXPORT html
<style>
.zebra-striping tbody tr:nth-child(odd) {
  background: #f2f2f2;
}
</style>
#+END_EXPORT

#+attr_html: :class zebra-striping
#+attr_css: :width 80%
| <l>                              |  <c>   |   <c>   |   <c>    |  <c>   |
| Program                          | [[https://github.com/GregorySchwartz][Github]] | [[http://hackage.haskell.org/user/GregorySchwartz][Hackage]] | [[https://www.stackage.org/][Stackage]] | [[https://hub.docker.com/u/gregoryschwartz][Docker]] |
|----------------------------------+--------+---------+----------+--------|
| birch-beer                       |   [[https://github.com/GregorySchwartz/birch-beer][★]]    |    [[http://hackage.haskell.org/package/birch-beer][★]]    |          |   [[https://hub.docker.com/r/gregoryschwartz/birch-beer][★]]    |
| clumpiness                       |   [[https://github.com/GregorySchwartz/clumpiness][★]]    |    [[http://hackage.haskell.org/package/clumpiness][★]]    |    [[https://www.stackage.org/lts-8.20/package/clumpiness-0.17.0.0][★]]     |        |
| convert-annotation               |   [[https://github.com/GregorySchwartz/convert-annotation][★]]    |    [[http://hackage.haskell.org/package/convert-annotation][★]]    |    [[https://www.stackage.org/lts-8.20/package/convert-annotation-0.5.0.1][★]]     |        |
| differential                     |   [[https://github.com/GregorySchwartz/differential][★]]    |    [[http://hackage.haskell.org/package/differential][★]]    |          |        |
| diversity                        |   [[https://github.com/GregorySchwartz/diversity][★]]    |    [[http://hackage.haskell.org/package/diversity][★]]    |    [[https://www.stackage.org/lts-8.20/package/diversity-0.8.0.2][★]]     |        |
| fasta                            |   [[https://github.com/GregorySchwartz/fasta][★]]    |    [[http://hackage.haskell.org/package/fasta][★]]    |    [[https://www.stackage.org/lts-8.20/package/fasta-0.10.4.2][★]]     |        |
| find-clumpiness                  |   [[https://github.com/GregorySchwartz/find-clumpiness][★]]    |    [[http://hackage.haskell.org/package/find-clumpiness][★]]    |    [[https://www.stackage.org/lts-9.0/package/find-clumpiness-0.2.1.3][★]]     |        |
| heatitup                         |   [[https://github.com/GregorySchwartz/heatitup][★]]    |    [[https://hackage.haskell.org/package/heatitup][★]]    |          |        |
| heatitup-complete                |   [[https://github.com/GregorySchwartz/heatitup-complete][★]]    |    [[https://hackage.haskell.org/package/heatitup-complete][★]]    |          |        |
| hierarchical-spectral-clustering |   [[https://github.com/GregorySchwartz/hierarchical-spectral-clustering][★]]    |    [[http://hackage.haskell.org/package/hierarchical-spectral-clustering][★]]    |          |        |
| integreat                        |   [[https://github.com/GregorySchwartz/integreat][★]]    |    [[http://hackage.haskell.org/package/integreat][★]]    |          |        |
| modify-fasta                     |   [[https://github.com/GregorySchwartz/modify-fasta][★]]    |    [[http://hackage.haskell.org/package/modify-fasta][★]]    |    [[https://www.stackage.org/lts-8.20/package/modify-fasta-0.8.2.3][★]]     |        |
| modularity                       |   [[https://github.com/GregorySchwartz/modularity][★]]    |    [[http://hackage.haskell.org/package/modularity][★]]    |          |        |
| random-tree                      |   [[https://github.com/GregorySchwartz/random-tree][★]]    |    [[http://hackage.haskell.org/package/random-tree][★]]    |    [[https://www.stackage.org/lts-8.20/package/random-tree-0.6.0.5][★]]     |        |
| rank-product                     |   [[https://github.com/GregorySchwartz/rank-product][★]]    |    [[http://hackage.haskell.org/package/rank-product][★]]    |    [[https://www.stackage.org/lts-8.20/package/rank-product-0.2.0.1][★]]     |        |
| spectral-clustering              |   [[https://github.com/GregorySchwartz/spectral-clustering][★]]    |    [[http://hackage.haskell.org/package/spectral-clustering][★]]    |          |        |
| too-many-cells                   |   [[https://github.com/GregorySchwartz/too-many-cells][★]]    |    [[http://hackage.haskell.org/package/too-many-cells][★]]    |          |   [[https://hub.docker.com/r/gregoryschwartz/too-many-cells][★]]    |
| tree-fun                         |   [[https://github.com/GregorySchwartz/tree-fun][★]]    |    [[http://hackage.haskell.org/package/tree-fun][★]]    |    [[https://www.stackage.org/lts-8.20/package/tree-fun-0.8.1.0][★]]     |        |

#+BEGIN_EXPORT html
<br></br>
#+END_EXPORT

* Our Team
:PROPERTIES:
:EXPORT_HUGO_SECTION:
:EXPORT_FILE_NAME: team
:END:

** Principal Investigator

#+BEGIN_EXPORT html
<div class="clearfix">

<img src="/img/people/gws-headshot.jpg" class="pull-left" style="border-radius:10%;margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

*** Gregory Schwartz

- Scientist at The Princess Margaret Cancer Centre
- [[https://medbio.utoronto.ca/faculty/schwartz][Assistant Professor at the University of Toronto]]
- Canada Research Chair in Bioinformatics and Computational Biology

#+BEGIN_EXPORT html
</div>

<br></br>
#+END_EXPORT

** Trainees

#+BEGIN_EXPORT html
<div class="clearfix">

<img src="/img/people/fatema-zohora-headshot.jpg" class="pull-left" style="border-radius:10%;margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

*** Fatema Zohora

- Postdoctoral Fellow

#+BEGIN_EXPORT html
</div>

<br></br>
#+END_EXPORT

#+BEGIN_EXPORT html
<div class="clearfix">

<img src="/img/people/christie-lau-headshot.jpg" class="pull-left" style="border-radius:10%;margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

*** Christie Lau

- Graduate Student

#+BEGIN_EXPORT html
</div>

<br></br>
#+END_EXPORT

** Staff

#+BEGIN_EXPORT html
<div class="clearfix">

<img src="/img/people/viet-hoang-headshot.png" class="pull-left" style="border-radius:10%;margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

*** Viet Hoang

- Bioinformatics Analyst

#+BEGIN_EXPORT html
</div>

<br></br>
#+END_EXPORT

#+BEGIN_EXPORT html
<div class="clearfix">

<img src="/img/people/ho-seok-lee-headshot.png" class="pull-left" style="border-radius:10%;margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

*** Ho Seok Lee

- Software Developer

#+BEGIN_EXPORT html
</div>

<br></br>
#+END_EXPORT

#+BEGIN_EXPORT html
<div class="clearfix">

<img src="/img/people/wendy-chen-headshot.jpg" class="pull-left" style="border-radius:10%;margin-right:5%;width:30%;height:auto" />
#+END_EXPORT

*** Wendy Chen

- Administrative Assistant

#+BEGIN_EXPORT html
</div>

<br></br>
#+END_EXPORT

* Open Positions
:PROPERTIES:
:EXPORT_HUGO_SECTION:
:EXPORT_FILE_NAME: positions
:END:

- We are hiring for all levels!

- From graduate student to postdoctoral fellow, we are looking for enthusiastic
  trainees who want to develop computational methods to understand
  treatment-resistance in cancer, cancer evolution, cancer heterogeneity,
  multi-omic analyses, and more!

- We are also looking for software developers and bioinformaticians that are
  excited about creating new types of data visualizations and interfaces to make
  our tools widely accessible.

*To apply to the Schwartz Lab*, please send your C.V., cover letter, and contact
information for three references to [[mailto:gregory.schwartz@uhnresearch.ca][gregory.schwartz@uhnresearch.ca]] for
consideration. We look forward to seeing your application!

#+BEGIN_EXPORT html
<br></br>
#+END_EXPORT

* Contact
:PROPERTIES:
:EXPORT_HUGO_SECTION:
:EXPORT_FILE_NAME: contact
:END:

- [[mailto:gregory.schwartz@uhnresearch.ca][gregory.schwartz@uhnresearch.ca]]

- [[https://github.com/GregorySchwartz][github]]

- [[https://www.linkedin.com/in/schwartzgregory][linkedin]]


#+BEGIN_EXPORT html
<br></br>
#+END_EXPORT

* News
:PROPERTIES:
:EXPORT_HUGO_SECTION: blog
:END:

** News
:PROPERTIES:
:EXPORT_FILE_NAME: _index
:END:

** Lab opening
SCHEDULED: <2021-10-04 Mon>
:PROPERTIES:
:EXPORT_FILE_NAME: lab_opening
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/high_park_cherry_blossoms.jpg"
:END:
The lab is opening in October! Join our team to help develop new methods and
algorithms to help us understand treatment-resistance in cancer and create new computational
tools to improve diagnosis and guide therapy. We are accepting applications for
all levels, from graduate student to postdoctoral fellow, as well as software
engineers and bioinformaticians.

** Christie Lau joins the lab as a rotation student!
SCHEDULED: <2021-10-20 Wed>
:PROPERTIES:
:EXPORT_FILE_NAME: christie_lau_joins_rotation
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/join_lab.png"
:END:
Christie Lau from the Department of Medical Biophysics joins the lab.

** Yuan Chang joins the lab as a rotation student!
SCHEDULED: <2021-11-26 Fri>
:PROPERTIES:
:EXPORT_FILE_NAME: yuan_chang_joins_rotation
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/join_lab.png"
:END:
Yuan Chang from the Department of Medical Biophysics joins the lab.

** Viet Hoang joins the lab as a Bioinformatics Analyst!
SCHEDULED: <2021-11-26 Fri>
:PROPERTIES:
:EXPORT_FILE_NAME: viet_hoang_joins_staff
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/join_lab.png"
:END:
Viet Hoang joins the lab as a Bioinformatics Analyst, welcome!

** The Schwartz Lab receives the CCS Challenge Grant
SCHEDULED: <2022-01-01 Sat>
:PROPERTIES:
:EXPORT_FILE_NAME: ccs_funding_2022-01-01
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/funding.png"
:END:
The Schwartz lab receives the Canadian Cancer Society Challenge Grant with
Catherine O'Brien to study drug-tolerant persisters.

** Christie Lau joins the lab as a PhD student!
SCHEDULED: <2022-01-04 Tue>
:PROPERTIES:
:EXPORT_FILE_NAME: christie_lau_joins
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/join_lab.png"
:END:
Christie Lau from the Department of Medical Biophysics joins the lab.

** The Schwartz Lab receives support from the DSI Research Software Development Support Program
SCHEDULED: <2022-04-01 Fri>
:PROPERTIES:
:EXPORT_FILE_NAME: dsi_software_developer_2022-04-01
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/funding.png"
:END:
The Schwartz lab receives support from the Data Science Institute Research
Software Development Support Program.

Featured in:
https://datasciences.utoronto.ca/advancing-data-science-discovery-via-software-development-support/

** Gregory presents at the Leukemia Academic Rounds
SCHEDULED: <2022-05-09 Mon>
:PROPERTIES:
:EXPORT_FILE_NAME: gregory_presents_2022-05-09
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/presentation.png"
:END:
Gregory presents at the Leukemia Academic Rounds.

** Christie presents at the DSI Student-Led Reproducibility Challenge
SCHEDULED: <2022-05-10 Tue>
:PROPERTIES:
:EXPORT_FILE_NAME: christie_dsi_reproducibility_2022-05-10
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/presentation.png"
:END:
Christie presents at the DSI Student-Led Reproducibility Challenge.

** Gregory presents at the Summer Student Seminar
SCHEDULED: <2022-06-14 Tue>
:PROPERTIES:
:EXPORT_FILE_NAME: gregory_presents_2022-06-14
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/presentation.png"
:END:
Gregory presents at the Summer Student Seminar.

** Fatema Zohora joins the lab as a Postdocal Fellow!
SCHEDULED: <2022-07-04 Mon>
:PROPERTIES:
:EXPORT_FILE_NAME: fatema_joins_2022-07-04
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/join_lab.png"
:END:
Fatema Zohora joins the lab as a Postdocal Fellow, welcome!

** Fatema is awarded the Next Generation Research Fellowship Fund
SCHEDULED: <2022-07-19 Mon>
:PROPERTIES:
:EXPORT_FILE_NAME: fatema_award_2022-07-19
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/funding.png"
:END:
Fatema is awarded the Next Generation Research Fellowship Fund, congrats!

** Ho Seok Lee joins the lab as a Software Developer!
SCHEDULED: <2022-09-12 Mon>
:PROPERTIES:
:EXPORT_FILE_NAME: david_joins_2022-09-12
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/join_lab.png"
:END:
Ho Seok Lee joins the lab as a Software Developer, welcome!

** Our tools were highlighted at the Data Sciences Institute!
SCHEDULED: <2022-10-04 Tue> 
:PROPERTIES:
:EXPORT_FILE_NAME: dsi_news_2022-10-04
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/newspaper.png"
:END:
Conor Klamann's work on an interactive TooManyCells interface is absolutely
incredible and featured by the Data Sciences Institute. Check out the article at
https://datasciences.utoronto.ca/building-data-science-software-to-help-the-fight-against-cancer/

** Gregory is awarded the Canada Research Chair in Bioinformatics and Computational Biology
SCHEDULED: <2022-11-17 Tue> 
:PROPERTIES:
:EXPORT_FILE_NAME: crc_news_2022-11-17
:EXPORT_HUGO_CUSTOM_FRONT_MATTER: :banner "img/news/funding.png"
:END:
Gregory is now the Canada Research Chair in Bioinformatics and Computational
Biology. [[https://www.uhnresearch.ca/news/research-chairs-awarded][Read more
about the award at UHN]] and
[[https://www.chairs-chaires.gc.ca/whats_new-quoi_de_neuf/2022/crc-2021-2-eng.aspx][the
Chairs website]].
